server/sandbox_run.go

```diff
@@ -464,27 +464,28 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	var ip string
 	// setup the network
 	if !hostNetwork {
-		if err = s.netPlugin.SetUpPod(netNsPath, namespace, kubeName, id); err != nil {
-			return nil, fmt.Errorf("failed to create network for container %s in sandbox %s: %v", containerName, id, err)
+		podNetwork := newPodNetwork(sb.Namespace(), sb.KubeName(), sb.ID(), netNsPath)
+		if err = s.netPlugin.SetUpPod(podNetwork); err != nil {
+			return nil, fmt.Errorf("failed to create network for container %s in sandbox %s: %v", sb.Name(), id, err)
 		}
 
-		if ip, err = s.netPlugin.GetContainerNetworkStatus(netNsPath, namespace, id, kubeName); err != nil {
-			return nil, fmt.Errorf("failed to get network status for container %s in sandbox %s: %v", containerName, id, err)
+		if ip, err = s.netPlugin.GetPodNetworkStatus(podNetwork); err != nil {
+			return nil, fmt.Errorf("failed to get network status for container %s in sandbox %s: %v", sb.Name(), id, err)
 		}
 
 		if len(portMappings) != 0 {
 			ip4 := net.ParseIP(ip).To4()
 			if ip4 == nil {
-				return nil, fmt.Errorf("failed to get valid ipv4 address for container %s in sandbox %s", containerName, id)
+				return nil, fmt.Errorf("failed to get valid ipv4 address for container %s in sandbox %s", sb.Name(), id)
 			}
 
 			if err = s.hostportManager.Add(id, &hostport.PodPortMapping{
-				Name:         name,
+				Name:         kubeName,
```

Comment:
sb.KubeName? Or sb.Name? 
Just to be consistent.

Reply:
@rajatchopra you're right, fixed; and removed that naming consistency commit since it no longer really makes sense.